Initial dataset examples (#7)
filecoin-project/slingshot: datasets.md

> 

Reviewer: ```suggestion
# Curated Datasets for the Slingshot Competition
```

> Slingshot’s aim for using curated datasets is to ensure meaningful data is stored and retrieved from the FileCoin Network.  The use-cases don’t need to be complex and can be proprietary in nature for applications.

Reviewer: ```suggestion
Slingshot’s aim for using curated datasets is to ensure meaningful data is stored and retrieved from the Filecoin Network.  The use-cases don’t need to be complex and can be proprietary in nature for applications.
```

> There are a wide variety of public data sets that can be leveraged for this challenge - a sampling is shown in the table below.   You are also welcome to submit another data set through the registration process.

Reviewer: ```suggestion
There are a wide variety of public data sets that can be leveraged for this challenge - a sampling is shown in the table below.  If you would like to use a dataset that you don't see listed here, please submit a PR to add the dataset to this table (i.e. through the registration process).
```
